frontend/views/components/sounds/Background.vue

```diff
@@ -6,20 +6,28 @@
 
 <script>
 import sbp from '@sbp/sbp'
+import { mapGetters } from 'vuex'
 import { MESSAGE_RECEIVE, MESSAGE_SEND } from '@model/contracts/shared/constants.js'
 
 export default ({
   name: 'BackgroundSounds',
+  computed: {
+    ...mapGetters(['isAppIdle'])
+  },
   created () {
     sbp('okTurtles.events/on', MESSAGE_RECEIVE, this.playMessageReceive)
     sbp('okTurtles.events/on', MESSAGE_SEND, this.playMessageSend)
   },
   methods: {
     playMessageReceive () {
-      this.$refs.msgReceive.play()
+      if (this.isAppIdle) {
+        this.$refs.msgReceive.play()
+      }
     },
     playMessageSend () {
-      this.$refs.msgSend.play()
+      if (this.isAppIdle) {
+        this.$refs.msgSend.play()
+      }
```

Comment:
Note that per the original issue it's OK to play the sound if the app is in the background and the user has recently moved the mouse.

So it might be worth using `idle-js` directly instead of this wrapper because that gives you access to `onHide` and `onShow`.

Reply:
Using `document.hidden`, I updated PR to make sound when the app is in background.